packages/round-manager/jest.config.ts

```diff
@@ -0,0 +1,34 @@
+import { Config } from "jest";
+
+const esModules = [
+  "@rainbow-me",
+  "@spruceid",
+  "github\\.com\\+gitcoinco\\+allo\\-indexer\\-client",
+  "wagmi",
+  "@wagmi",
+  "@gitcoinco",
+];
+
+module.exports = {
+  preset: "ts-jest/presets/js-with-ts",
+  testEnvironment: "<rootDir>/src/tests/fixedEnvironment.js",
+  globals: {
+    Uint8Array: Uint8Array,
+    ArrayBuffer: ArrayBuffer,
+    "ts-jest": {
+      diagnostics: {
+        exclude: ["**"],
```

Comment:
what does this do? does it disable type checking for tests?

Reply:
Yes. For some reason, typechecking is behaving differently in Jest and in the rest of the project, most likely because of the different environments. I'll try enabling it again and fixing the errors after I test all the onchain interactions.

Reply:
The issue only exists in jest, and after some investigation I'm fairly confident it's due to Jests's module handling (CommonJS/ES) and other intricacies. We can't even ignore the error because it wouldn't pass tsc/eslint because it's "ignoring an error that isn't there" for normal tsconfig. The errors happen when using the `Transition` component from headless UI, and setting the `as` props to React's Fragment.